### PR TITLE
Update Avatars

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR changes the margins on avatars since we removed them from core.

### How do I test this PR?
1. Look at the margins around the avatars in the sim and see if they look alright.
Caveat: The "missing image" avatar is smaller than an actual image so the margins around it look off ... but they're not.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [n/a] Closes [DEV-XXX](#url)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ const GreyH6 = styled(({ theme }) => ({
 const StyledCardContent = styled(({ theme }) => ({
   alignItems: 'center',
   marginBottom: theme.sizing.baseUnit,
+  paddingVertical: 0,
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js]: Why are we adding padding here?

Comment (richarddubay) [reply packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js]: @redreceipt Why do you hate padding? ;)

Just kidding. I removed it. I have no idea why that was there.

**Review by redreceipt — APPROVED** [review]
